packages/apsara-ui/package.json

```diff
@@ -1,6 +1,6 @@
 {
     "name": "@goto-company/apsara",
-    "version": "0.10.0",
```

Comment:
let's also update apsara-icons and version in lerna json

Reply:
Done